app/Http/Controllers/JobController.php

```diff
@@ -68,6 +67,8 @@ public function show($id)
         } else {
             $similarJobs = $tag->jobs;
         }
+        $jobCurrent = Job::where('id', $id)->get();
```

Comment:
```suggestion
        $jobCurrent = Job::findOrFail($id);
```
Dùng cách này cho tiện nha

Reply:
Em dùng where để kết quả trả về là 1 collection, khi đó mới sử dụng đc hàm diff bên dưới ạ. $similarJobs = $similarJobs->diff($jobCurrent);